src/main/java/org/opentripplanner/model/GroupOfRoutes.java

```diff
@@ -0,0 +1,55 @@
+package org.opentripplanner.model;
+
+import org.opentripplanner.model.base.ToStringBuilder;
+
+/**
+ * OTP model for NeTEx GroupOfLines. Not used for GTFS at the moment.
```

Comment:
You should add some documentation on what purpose this class play or at least give one example use-case. Why do we need this class?

Reply:
Updated documentation